lib/contributor.rb

```diff
@@ -0,0 +1,11 @@
+class Contributor
+  def initialize(contributor)
+    @contributor = contributor
+  end
+
+  def add
+    File.open('CONTRIBUTORS.md', 'a+') do |f|
+      f.puts "- [@#{@contributor}] (https://github.com/#{@contributor})"
+    end
+  end
+end
```

Comment:
Happy with appending chronologically, or want to consider alphanumeric sorting? I guess the former would result in less lines changed (and less TODO)